cds/modules/deposit/static/templates/cds_deposit/angular-schema-form/uiselect.html

```diff
@@ -21,6 +21,7 @@
                 {{select_model.selected.name}}
             </ui-select-match>
       <ui-select-choices
+        minimum-input-length="3"
```

Comment:
In the end make sense also to set a value.
We should maintain consistency for every field that is using autocomplete.